Rename `element-animal` to `element-subject` (#13)
datajoint/workflow-animal: README.md

This workflow builds a workflow to save the information of lab management and animal management, using the following datajoint elements
+ [element-lab](https://github.com/datajoint/element-lab)
+ [element-subject](https://github.com/datajoint/element-subject)

This repository provides demonstrations for:
Set up a workflow using different elements (see [workflow_animal/pipeline.py](workflow_animal/pipeline.py))

## Workflow architecture

> The lab and animal management workflow presented here uses components from two DataJoint elements, element-lab, and element-subject assembled together to a functional workflow.

Reviewer: ```suggestion
The Lab and Animal Management Workflow presented here uses two DataJoint Elements: element-lab and element-subject assembled into a functional workflow.
```

Author: Thanks!
